index.js

```diff
@@ -49,6 +58,15 @@ function formatDate(d) {
   return d.getFullYear() + pad2(d.getMonth() + 1) + pad2(d.getDate());
 };
 
+function sanitizeFileName(filename) {
```

Comment:
We just so happened to have merged some similar code in ojos: https://github.com/movableink/ojos/pull/585/files#diff-19dd784a10c47d0cffaad6f5e3e13070R98

This may be easier as a regex-replace?

```
return filename.replace(/[^A-Za-z0-9_\-\.]/, ()=>'');
```

Reply:
love it.